lib/MusicBrainz/Server/Report/CDTOCDubiousLength.pm

```diff
@@ -0,0 +1,42 @@
+package MusicBrainz::Server::Report::CDTOCDubiousLength;
+use Moose;
+
+with 'MusicBrainz::Server::Report::CDTOCReport';
+
+sub table { 'cd_toc_dubious_length' }
```

Comment:
I guess that was the missing bit.

Reply:
I think the main issue was the link generated from the reports index was wrong, too :D But a bit of everything maybe :)